core/vm/actor/builtin/storage_power/storage_power_actor_state.cpp

```diff
@@ -3,26 +3,35 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "vm/actor/builtin/storage_power/storage_power_actor.hpp"
+#include "vm/actor/builtin/storage_power/storage_power_actor_state.hpp"
 
 #include "power/impl/power_table_impl.hpp"
 #include "vm/exit_code/exit_code.hpp"
 #include "vm/indices/indices.hpp"
 
 namespace fc::vm::actor::builtin::storage_power {
 
-  const power::Power StoragePowerActor::kMinMinerSizeStor =
-      100 * (primitives::BigInt(1) << 40);
-
-  const size_t StoragePowerActor::kMinMinerSizeTarg = 3;
+  StoragePowerActorState::StoragePowerActorState(
+      std::shared_ptr<Indices> indices,
+      std::shared_ptr<crypto::randomness::RandomnessProvider>
+          randomness_provider)
+      : indices_(std::move(indices)),
+        randomness_provider_(std::move(randomness_provider)),
+        total_network_power_(0),
+        power_table_(std::make_unique<power::PowerTableImpl>()),
+        claimed_power_(std::make_unique<power::PowerTableImpl>()),
+        nominal_power_(std::make_unique<power::PowerTableImpl>()),
```

Comment:
```suggestion
        nominal_power_(std::make_unique<power::PowerTableHamt>()),
```
Also for two lines above

Reply:
Not yet, the state should be reworked (up-to date specs-actor, CBOR-serialisation, etc.)